src/test/kotlin/apply/ui/api/CheaterRestControllerTest.kt

```diff
@@ -63,10 +63,10 @@ internal class CheaterRestControllerTest : RestControllerTest() {
 
     @Test
     internal fun `부정행위자를 추가한다`() {
-        every { cheaterService.save(cheatedApplicant.id) } just Runs
+        every { cheaterService.save(cheatedApplicant.email) } just Runs
 
         mockMvc.post("/api/cheaters") {
-            param("applicantId", cheatedApplicant.id.toString())
+            param("email", cheatedApplicant.email.toString())
```

Comment:
```suggestion
            param("email", cheatedApplicant.email)
```
toString은 빠져도 되겠네요!! ㅎㅎㅎ

Reply:
앗 그러고 아론이 작업하신 부분은 아니지만, 
말 나온김에 여기 부분 반영하면서 
이 클래스 내에서만 제거되지 않은 internal을 제거해주시면 정말 좋을 것 같습니다 ㅎㅎㅎ 🙇‍♂️

Reply:
제거했습니다 👍